pkg/didmethod/peer/resolver.go

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package peer
+
+import (
+	errors "golang.org/x/xerrors"
+)
+
+// DIDResolver resolver
+type DIDResolver struct {
+	store *DIDStore
+}
+
+// NewDIDResolver new Peer DID resolver instance
+func NewDIDResolver(store *DIDStore) *DIDResolver {
+	return &DIDResolver{store: store}
+}
+
+// Read implements DidMethod.Read function
```

Comment:
Can we please provide a fuller description?

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/compare/476126814875769cf2630645a8567f84753b0219..0dc9081c1b8917fad36c8e7230cbf4a4f67693f4#diff-85c3d889e3df4373e3188d4932b12f3fR23) the description and added link to the spec.